exercises/spiral-matrix/example.js

```diff
@@ -0,0 +1,27 @@
+class SpiralMatrix {
+
+  static ofSize(size) {
+    const spiral = [];
+    for (; spiral.length < size; spiral.push([]));
+
+    const totalNumbers = size * size;
```

Comment:
If you like, you can use the new exponentiation operator:

```javascript
const totalNumbers = size**2;
``` 

Node 7 should support it.